operator/pkg/ingress/ingress_reconcile_test.go

```diff
@@ -526,6 +526,47 @@ func TestReconcile(t *testing.T) {
 		require.Empty(t, ingress.Status.LoadBalancer.Ingress, "Loadbalancer status of Ingress should be reset")
 	})
 
+	t.Run("Reconcile of dedicated Cilium Ingress with loadbalancer class will create the dedicated loadbalancer service with the specified class", func(t *testing.T) {
```

Comment:
Could you please also add a negative test - that is, that a shared Ingress with loadbalancer class set doesn't do anything? I can see from the code that it won't for now, but better to lock this behavior in with a test while you're here.

Reply:
done!